scripts/build_cmake.bash

```
@@ -2,6 +2,8 @@
 
 # Helper script that builds all the packages using vanilla CMake.
 
+set -e  # Needed to ensure cmake failures propagate up
```

this was why our "vanilla cmake" workflows were swallowing build errors! 🤦🏻

---

lol derp :/